sinks/api/supported_labels.go

```diff
@@ -15,13 +15,15 @@
 package api
 
 const (
-	LabelPodId         = "pod_id"
-	LabelPodName       = "pod_name"
-	LabelPodNamespace  = "pod_namespace"
-	LabelContainerName = "container_name"
-	LabelLabels        = "labels"
-	LabelHostname      = "hostname"
-	LabelResourceID    = "resource_id"
+	LabelPodId               = "pod_id"
+	LabelPodName             = "pod_name"
+	LabelPodNamespace        = "pod_namespace"
+	LabelContainerName       = "container_name"
+	LabelLabels              = "labels"
+	LabelHostname            = "hostname"
+	LabelResourceID          = "resource_id"
+	LabelComputeResourceID   = "compute.googleapis.com/resource_id"
```

Comment:
We need labels of our metrics to be in a format compatible with google cloud autoscaler. So, we need to have prefix "compute.googleapis.com" for them. I'm afraid we cannot change the labels later when pushing to GCM. Maybe @tkulczynski can comment on this.